packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -95,8 +108,8 @@ jobs:
         uses: actions/upload-artifact@v4
         if: steps.check-for-e2e.outputs.has-e2e == 'true' && steps.run-e2e-tests.outcome != 'success'
         with:
-          name: cypress-videos
-          path: cypress/videos
+          name: playwright-report
+          path: playwright-report/
```

Comment:
Hmm the e2e [version resolver](https://grafana.com/developers/plugin-tools/e2e-test-a-plugin/ci#the-e2e-versions-action) is probably the most valuable part of the e2e experience IMO, so would be nice if we could find a way to integrate it with this workflow. 

What do you think about the following: 

After the `Stop grafana docker` step, we add a new step that uploads the dist folder as an artifact. Then we add a new job that uses the `e2e-version` Action to resolve a matrix of Grafana images to test against (see [here](https://github.com/grafana/plugin-tools/blob/plugin-e2e/add-new-selectors/.github/workflows/playwright.yml/#L13-L24)). Then we run e2e test for the different images in parallel. Similar to how we do it [here](https://github.com/grafana/plugin-tools/blob/plugin-e2e/add-new-selectors/.github/workflows/playwright.yml/#L28-L77), but unlike in this example we don't need to build the BE and FE again - we can just pull the artifacts that were uploaded in the previous job. Lastly, if all previous steps and jobs succeeded, there's a new job that pulls artifacts again and run steps for signing, packaging and achiving the build. WDYT?

Reply:
I think that could be a good addition but it feels more like a feature addition as currently we only run it with cypress against a single version of Grafana. Do you mind if we land this first then figure out the `version resolver` in a separate PR?

Reply:
Yes of course, we can handle this in an upcoming PR. 👍